service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class EmailNotificationHelperTest {
+
+    @Test
+    void subjectLineShouldBeEmpty() {
```

Comment:
Not sure what's this test is doing:

* we have got two separates method in the class
* The method has got some logic with ? : or if statemetns and we are just checking if the subjectLine is not empty.

I don't think this test is good enough.

Reply:
updated